components/badge/badge.test.tsx

```diff
@@ -50,4 +50,14 @@ describe("Badge", () => {
     .create(<Badge><StringComponent /></Badge>)
     .toJSON()).toMatchSnapshot()
   });
+
+  it("can have onClick event", () => {
+    const foo = () => {
```

Comment:
I would love to see this inlined instead of using a random variable here :)

Reply:
using arrow functions inline decrease performance and we have a rule in tslint to enforce nobody use it.